config/opal_setup_wrappers.m4

```diff
@@ -130,6 +130,12 @@ AC_DEFUN([OPAL_SETUP_WRAPPER_INIT],[
                   [enable rpath/runpath support in the wrapper compilers (default=yes)])])
     AS_IF([test "$enable_wrapper_rpath" != "no"], [enable_wrapper_rpath=yes])
     AC_MSG_RESULT([$enable_wrapper_rpath])
+
+    AC_ARG_ENABLE([new-dtags],
```

Comment:
Probably need `wrapper` in the option name somewhere, to mirror the form of `--enable-wrapper-rpath` (because this option *only* affects the wrapper compiler flags, right?).

Idle question: should we call this option `--enable-new-dtags` or `--enable-runpath`?  "New dtags" is that the GNU linker calls it, but the generic name is "runpath".  I.e., I'm wondering if `--enable-runpath` would be a bit more user friendly...?  (this is an open question -- I'm open to opinions here)